dali/operators/sequence/optical_flow/optical_flow.h

```diff
@@ -41,6 +41,7 @@ struct backend_to_compute<GPUBackend> {
 
 const std::string kPresetArgName = "preset";                               // NOLINT
```

Comment:
A general remark: `std::string` constants are not a good idea - but if they are used, they should be declared `static` or placed in an anonymous namespace - otherwise the program will fail to link if this header is included in multiple translation units.

Reply:
Done